internal/resources/cluster/cluster_full_name.go

```diff
@@ -14,7 +14,7 @@ import (
 
 var ClusterFullname = &schema.Schema{
 	Type:        schema.TypeList,
-	Description: "The schema for cluster iam policy full name",
+	Description: "The schema for cluster full name",
```

Comment:
I am not sure of this change, do check if the models for cluster full name of IAM policy matches the cluster full name. As far as I remember, the spec was different. Please confirm before making this change and reusing it in source secret to any other place. Also, if this is reusable, then this schema description change should be shown in docs. Please regenerate the docs in that case. 
cc: @ishangupta-ds